### PR TITLE
Fix deployment issues on lsst-lsp-int-dax

### PR DESCRIPTION
**Pull request by kennywlo**
Pls review.

### CODE REVIEW

File: kube/int/create.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -ex
-DAX_NAMESPACE=${DAX_NAMESPACE:-'dax-int'}
+DAX_NAMESPACE=${DAX_NAMESPACE:-'lsst-lsp-int-dax'}
 
 # kubectl create -f dax-imgserv-datasets-volume.yaml
 # kubectl create -f dax-imgserv-datasets-claim.yaml --namespace $DAX_NAMESPACE
```
Comment (cbanek) [inline kube/int/create.sh]: Follow my pattern that you reviewed yesterday to create the volumes in another script.  Also, fix the names.

Comment (kennywlo) [reply kube/int/create.sh]: Done

**Comment by cbanek** [comment]
Also you misspelled lsp in your title of the pull request

File: kube/int/destroy.sh [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -x
-DAX_NAMESPACE=${DAX_NAMESPACE:-'dax-int'}
+DAX_NAMESPACE=${DAX_NAMESPACE:-'lsst-lsp-int-dax'}
 
 kubectl delete ingress dax-imgserv-ingress --namespace $DAX_NAMESPACE
 kubectl delete service dax-imgserv-service --namespace $DAX_NAMESPACE
```
Comment (cbanek) [inline kube/int/destroy.sh]: You're still not following the pattern with destroy_volumes.  Move these lines there

Comment (kennywlo) [reply kube/int/destroy.sh]: How about now?

Comment (cbanek) [reply kube/int/destroy.sh]: Well this is better, but you didn't fix the names of the volume, so your pvc doesn't point to the right volume, you missed the rename there (and in the names of the yaml files)

File: kube/int/destroy_volumes.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -x
+DAX_NAMESPACE=${DAX_NAMESPACE:-'lsst-lsp-int-dax'}
+
+kubectl delete pvc dax-imgserv-datasets-claim --namespace $DAX_NAMESPACE
+kubectl delete pv dax-int-imgserv-datasets-volume
```
Comment (cbanek) [inline kube/int/destroy_volumes.sh]: Wrong pv and pvc name.

**Pull request merged**